src/app/qml/DownloadPage.qml

```diff
@@ -250,11 +266,13 @@ Page {
                         releases.variant
                 }   
             }
-            
         },
         State {
             name: "failed_verification_no_drives"
             when: releases.variant.status === Units.DownloadStatus.Failed_Verification && drives.length <= 0
+                        StateChangeScript {
+                script: console.error("failed_verification_no_drives")
```

Comment:
Is this intentional?  Or just a leftover? We should definitely inform the user visually and not into the stderr.